addons/dragging/functions/fnc_carryObject.sqf

```diff
@@ -63,7 +63,7 @@ _unit setVariable [QGVAR(ReleaseActionID), [
 if (_target isKindOf "CAManBase") then {
     [localize LSTRING(Drop), "", ""] call EFUNC(interaction,showMouseHint);
 } else {
-    [localize LSTRING(Drop), "", localize LSTRING(LowerRaise)] call EFUNC(interaction,showMouseHint);
+    [localize LSTRING(Drop), "", localize LSTRING(RaiseLowerRotate)] call EFUNC(interaction,showMouseHint);
```

Comment:
```suggestion
    [localize LSTRING(Drop), "", LLSTRING(LowerRaise), [["control", LLSTRING(rotate)]]] call EFUNC(interaction,showMouseHint);
```
The advantage of doing this now is we don't have to drop our existing translations

Reply:
It's hideous, though